### PR TITLE
 Issue #500

### PR DESCRIPTION
**Pull request by alex85vol**
need review.

### CODE REVIEW

**Comment by stoune** [comment]
Why we need this? :-1:

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
+
 # CodeReviewTestRepository
+# Change 1
+=============
+# Change 2
```
Comment (stoune) [inline README.md:5]: Why we need this topic?

Comment (tsypuk) [inline README.md:5]: What this changes are doing ?

Comment (alex85vol) [inline README.md:5]: Cause I need it.

**Comment by stoune** [comment]
Looks good. :+1:

**Comment by tsypuk** [comment]
LGTM +1

**Pull request merged**